server/root.ts

```diff
@@ -547,31 +565,33 @@ const root = {
   },
 
   DeclineHangoutRequest: async params => {
```

Comment:
refactored. don't need the entire userLimitedObject to remove it from the users' sent/received_hangout_requests array. just the email is okay